# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Runtime.cpp
# Commit: e862899dca3f
# Full Hash: e862899dca3f252a8fe3c7be10d83d79dad328c5
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2018-05-16 18:32:17
# Regressor Bug: 1461751
# File Overlap Count: 2
# Description:
#   Bug 1461751 - Simplify module resolve hook to be a function pointer r=luke r=baku
# ==============================================================================

diff -r 1076e77e7b5c -r e862899dca3f js/src/vm/Runtime.cpp
--- a/js/src/vm/Runtime.cpp	Tue Apr 24 16:19:51 2018 +0200
+++ b/js/src/vm/Runtime.cpp	Wed May 16 11:59:09 2018 +0100
@@ -174,7 +174,8 @@
     performanceMonitoring_(),
     stackFormat_(parentRuntime ? js::StackFormat::Default
                                : js::StackFormat::SpiderMonkey),
-    wasmInstances(mutexid::WasmRuntimeInstances)
+    wasmInstances(mutexid::WasmRuntimeInstances),
+    moduleResolveHook()
 {
     JS_COUNT_CTOR(JSRuntime);
     liveRuntimesCount++;